app/Http/Controllers/Api/V1/ActivityTypeController.php

```diff
@@ -224,7 +216,7 @@ public function update(UpdateActivityType $request, Organization $suborganizatio
      *
      * Remove the specified activity type.
      *
```

Comment:
Need to add "$suborganization" as "urlParam".

Reply:
Done!